decentralized_leader_follower_control:
  Kp_x : 2.5
  Kp_y : 0
  Kp_phi : 0
  lin_vel_max : 0.2
  ang_vel_max : 0.3